vibi-dpu/src/github/auth.rs

```diff
@@ -78,6 +78,7 @@ pub async fn fetch_access_token(installation_id: &str) -> Option<GithubAuthInfo>
     }
     let mut gh_auth_info = gh_auth_info_opt.expect("Uncaught error in gh_auth_info_opt");
     save_github_auth_info_to_db(&mut gh_auth_info);
+    gh_auth_info.save_to_file();
```

Comment:
Since this is not essential to the responsibilities of the `fetch_access_token` function, I thought it would make sense to run the `save_to_file` function asynchronously. (File I/O is high-cost operation; we don't want that to slow down the performance of the application).

LOGAF: 5/10